command/commands/user.py

```diff
@@ -166,13 +167,14 @@ def delete_helper(self, user_id, slack_id):
         """
         Delete user from database.
 
-        Delete user with slack_id from database if user with user_id has
-        admin permission level.
+        Delete user with ``slack_id`` from database if user with ``user_id``
+        has admin permission level. **Note**: deleting yourself is entirely
```

Comment:
Not sure why you would ever want to delete yourself. Can you tell me a specific case of where this is useful?

Reply:
I think that's the point of his comment - it's rather unintuitive to expect to be able to delete yourself, yet because of the way we've designed things it is actually possible, so be careful not to do it. (Though in that case, I agree - this could be reworded slightly, to sound more like a warning rather than just a random parenthetical comment.)